tests/test_mcts.py

```diff
@@ -7,14 +7,36 @@ class TestMCTS:
     def test_tictactoe_first_step(self):
         policy, _ = mcts(
             start_node=MCTSNode(
-                state=TicTacToe.get_initial_board(3, 3),
-                player=0,
-                root_player=0,
+                state=TicTacToe.get_initial_board(3, 3), player=0,
             ),
             rollout=True,
             game=TicTacToe,
-            n_iter=1000,
+            n_iter=10000,
             c_puct=5.0,
+            dirichlet_eps=0,
+            dirichlet_conc=1,
         )
-        print(policy)
         assert torch.argmax(policy).item() == 4  # Middle
+
+    def test_mcts_net(self):
+        # Just check that it runs for now
+        def dummy_net(inp):
+            return torch.ones(3, 3) / 9, torch.tensor(0)
+
+        mcts(
+            start_node=MCTSNode(
+                state=TicTacToe.get_initial_board(3, 3), player=0,
+            ),
+            rollout=False,
+            net=dummy_net,
+            game=TicTacToe,
+            n_iter=1000,
+            c_puct=5.0,
+        )
+
+    def test_self_play(self):
```

Comment:
`test_self_play_runs`? you might want to separate self_play into several functions and test their sub-functionality instead of having everything in one function like this... it's not a very good test :P

Reply:
I get what you mean but it's hard to do because the functions are internal. I'll try to think of some more tests, and rename this.

Reply:
Renamed, will create issue for more tests